code/game/atoms_movable.dm

```diff
@@ -61,7 +63,91 @@
 			return FALSE
 	return ..()
 
+/atom/movable/proc/start_pulling(atom/movable/AM,gs)
+	if(!AM || !src)
+		return FALSE

```

Comment:
Change this to just !QDELETED(AM)

Reply:
are you sure, that !QDELETED(AM), not QDELETED(AM) ?